libraroo-backend/src/catalog-books/catalog-book-mapper.ts

```diff
@@ -17,7 +17,7 @@ export class CatalogBookMapper {
   static toCatalogBookDb(book: BookItem): Partial<CatalogBook> {
     return {
       authors: book.volumeInfo.authors,
-      title: book.volumeInfo.title,
+      title: `${book.volumeInfo.title} ${book.volumeInfo.subtitle}`,
```

Comment:
If subtitle is undefined then there will be a unnececerry space at the end of the title